src/components/Header/Header.jsx

```diff
@@ -127,13 +133,14 @@ const Web3Status = () => {
 
 const Web3StatusInner = ({ onWalletModalToggle = noop }) => {
   const { t } = useTranslation();
-  const { connectedAddress, connector, error } = useActiveStarknetReact();
-  if (connectedAddress) {
+  const { error } = useAccountDetails();
```

Comment:
you can remove line 136 since account details hook is not returning error

Reply:
Done